src/components/layouts/pages/inspect/pdfDocument/PdfDocument.js

```diff
@@ -90,11 +91,23 @@ class PdfDocument extends React.PureComponent {
 
                 objectOfErrors = { ...nextStepObject };
             });
-            return findAllMcid(objectOfErrors);
+            return findAllMcid(objectOfErrors, page);
```

Comment:
We can't get page from context which is a path to structure element in structure tree. In this case, `page` will always be `-1`. So, we don't need to pass it to `findAllMcid` function

Reply:
Removed useless page